controllers/actions.github.com/autoscalinglistener_controller.go

```diff
@@ -246,65 +245,6 @@ func (r *AutoscalingListenerReconciler) Reconcile(ctx context.Context, req ctrl.
 	return ctrl.Result{}, nil
 }
 
-// SetupWithManager sets up the controller with the Manager.
```

Comment:
Moved to be the last method on the reconciler, just like for rest of the reconcilers